src/quo2/components/buttons/dynamic_button.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo2.components.buttons.dynamic-button
+  (:require [quo.react-native :as rn]
+            [reagent.core :as reagent]
+            [status-im.i18n.i18n :as i18n]
+            [quo2.components.icon :as icon]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]))
+
+(defn- in?
+  "true if collection contains element"
+  [collection element]
+  (some #(= element %) collection))
+
+(defn- get-button-color [type pressed? customization-color]
+  (if (in? '(:jump-to :mention) type)
```

Comment:
could be just `(#{:jump-to :mention} type)`

Reply:
yes it looks better, will replace in https://github.com/status-im/status-mobile/issues/14071. Thank you